vnext/Microsoft.ReactNative/JsiApi.cpp

```diff
@@ -0,0 +1,497 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include "JsiApi.h"
+#include "JsiPreparedJavaScript.g.cpp"
+#include <crash/verifyElseCrash.h>
+
+// facebook::jsi::Runtime hides all methods that we need to call.
+// We "open" them up here by redeclaring the private and protected keywords.
+#define private public
+#define protected public
+#include "jsi/jsi.h"
```

Comment:
>#inclu [](start = 0, length = 6)

Shouldn't we `#undef private` and public (we declare private below, so that would override our code too :)

Reply:
Great suggestion! I will do that.

Reply:
Great suggestion! I have added it.

---
In reply to: [454755479](https://github.com/microsoft/react-native-windows/pull/5461#discussion_r454755479) [](ancestors = 454755479)